docker-compose.yaml

```diff
@@ -0,0 +1,64 @@
+version: "3"
+services:
+
+  # this could be enabled once the server container is added
+  server:
+    build:
+      .
+    depends_on:
+      - database
+      - redis
+    ports:
+      - "3000:3000"
+    environment:
+        DEBUG: "speckle:*"
+    env_file:
+      - .env
+    
+  database:
+    image: "postgres" # use latest official postgres version
+    env_file:
+      - .env # configure postgres
+    volumes:
+      # persist data even if container shuts down
+      - speckle-postgres-data:/var/lib/postgresql/data/
+      # this mounts the initdb folder to the db container
+      # .sql .sh files are executed from this folder in alphabetical order
+      - ./initdb:/docker-entrypoint-initdb.d  
+    ports:
+      - "5432:5432"
+
+  pgadmin:
+    image: dpage/pgadmin4
+    environment: 
+        PGADMIN_DEFAULT_EMAIL: "${PGADMIN_EMAIL}"
+        PGADMIN_DEFAULT_PASSWORD: "${PGADMIN_PASSWORD}"
+    ports: 
+        - "16543:80"
+    depends_on: 
+        - database
+
+  redis:
+    image: "redis"
```

Comment:
same as postgres, could be the alpine image

Reply:
Yup.